app/views/layouts/application.html.erb

```diff
@@ -44,6 +44,10 @@
 
     <%= yield :primary_navigation_content %>
 
+    <div class="govuk-width-container">
+      <%= yield :breadcrumbs %>
+    </div>
```

Comment:
It might be nice to use [content_for?](https://api.rubyonrails.org/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for-3F) here so we don't bloat the page with unnecessary empty `<div>`s.

Bonus points if you do the same for the `before_content` wrapper too ⭐

```suggestion
    <% if content_for?(:breadcrumbs) %>
      <%= tag.div yield(:breadcrumbs), class: "govuk-width-container" %>
    <% end %>
```